internal/config/image/cuda_image.go

```diff
@@ -218,14 +225,25 @@ func (i CUDA) OnlyFullyQualifiedCDIDevices() bool {
 	return hasCDIdevice
 }
 
-const (
-	deviceListAsVolumeMountsRoot = "/var/run/nvidia-container-devices"
-)
+// VisibleDevicesFromMounts returns the set of visible devices requested as mounts.
+func (i CUDA) VisibleDevicesFromMounts() []string {
```

Comment:
Should there be a corresponding ` VisibleDevicesFromEnvVar()` for symmetry.

Reply:
I renamed this. Note that `DevicesFromEnvvars` is a bit more general as it's also used for swarm support.